rover_msgs/MosfetCmd.lcm

```diff
@@ -1,18 +1,18 @@
 package rover_msgs;
 
 struct MosfetCmd {
-	// TODO: name device constants
-	const int8_t DEV0 = 0;
-	const int8_t DEV1 = 1;
-	const int8_t DEV2 = 2;
-	const int8_t DEV3 = 3;
-	const int8_t DEV4 = 4;
-	const int8_t DEV5 = 5;
-	const int8_t DEV6 = 6;
-	const int8_t DEV7 = 7;
-	const int8_t DEV8 = 8;
-	const int8_t DEV9 = 9;
-
+    const int8_t rLed = 0;
+    const int8_t gLed = 1;
+    const int8_t bLed = 2;
+    const int8_t Laser = 3;
+    const int8_t UVLED = 4;
+    const int8_t whiteLED = 5;
+    const int8_t uvBulb = 6;
+    const int8_t nichWire0 = 7;
```

Comment:
why is this in here and there is also a heater lcm?

Reply:
i can remove heaters from mosfet cmd.

Reply:
yah would be preferable if you have a separate heater lcm